src/new-client/playwright.config.ts

```diff
@@ -21,7 +21,7 @@ export default defineConfig({
   // Let's try this for a bit with the goal of eliminating flakiness, but add in retries if we have to.
   retries: process.env.CI ? 2 : 0,
   // TODO: Playwright init defaults to opt out of parallel tests on CI. We should confirm if we agree with that
-  workers: process.env.CI ? 1 : undefined,
+  workers: process.env.CI ? 4 : undefined,
```

Comment:
Ooh good shout addressing this - hopefully might speed things up a bit! Could we remove the comment above it too if we think it's now resolved? 

Also, is there any particular reason for landing on 4? According to [the docs](https://playwright.dev/docs/api/class-testconfig#test-config-workers), if we leave this undefined it'll default to 50% of available CPU cores. We could even whack it right up to `'100%'` or 80% or something and see how it copes?